packages/truss/src/methods.ts

```diff
@@ -16,11 +16,13 @@ export function newMethod(abbr: UtilityName, defs: Properties): UtilityMethod {
 /**
  * Given a single abbreviation (i.e. `mt`) and a property name (i.e. `marginTop`), returns the
  * TypeScript code for a `mt` utility method that accepts a user-provided value of the prop to set.
+ * Use `extraDefs` for additional properties to set.
  *
  * I.e. `Css.mt(someValue).$`
  */
-export function newParamMethod(abbr: UtilityName, prop: keyof Properties): UtilityMethod {
-  return `${comment({ [prop]: "value" })} ${abbr}(value: Properties["${prop}"]) { return this.add("${prop}", value); }`;
+export function newParamMethod(abbr: UtilityName, prop: keyof Properties, extraDefs: Properties = {}): UtilityMethod {
```

Comment:
Suggestion: Maybe rename `extraDefs` to `extraProperties`. The "def" makes me think too much of "method definition", which is also something Truss does, but is not what this is.

I guess if it was "extra defaults" that would be less ambiguous, but I think `extraProperties` is probably best.

Reply:
`extraProperties` sounds better, thanks 👍